diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -45,8 +46,16 @@ open class DiktatJavaExecTaskBase @Inject constructor(
             if (diktatExtension.debug) {
                 add("--debug")
             }
-            add(diktatExtension.inputs.files.joinToString { it.path })
+            diktatExtension.inputs.files.forEach {
+                val pattern = project.trimRootDir(it.path)
```

Comment:
Are you sure, that path (it) here is absolute path?

Reply:
I believe, yes, this is from gradle filesystem API, all these methods use [project.file](https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:file(java.lang.Object)) under the hood.